### PR TITLE
T1059423: Change the DateBox demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/DateBox/Overview/description.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
...
+Specifies whether the DateBox allows users to select one of the following types:
+
+  - *"date"*    
+  Users can select the date from the calendar, or they can type in their own values.
+
+  - *"time"*    
+  Users can select from the time slots that lay in range from 12 AM to 11:30 PM with a step of 30 minutes, or they can type in their own values. Specify the [interval](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#interval) property to set the different time step.
+
+  - *"datetime"*    
+  Users can select both the date from the calendar and the time. The **Today** button is available in the date picker.
```
Comment (albertov05) [inline JSDemos/Demos/DateBox/Overview/description.md]: In the demo, I see that the datetime editor shows a calendar and a clock with editors. Can we say something like the following?

`Users can select the date from the calendar and the time from the spin and dropdown editors. They can also choose the **Today** button.`

File: JSDemos/Demos/DateBox/Overview/description.md [hunk 2]
```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
...
 
 - [displayFormat](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#displayFormat)        
 A date/time display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter.
```
Comment (albertov05) [inline JSDemos/Demos/DateBox/Overview/description.md:19]: This line is fine, but I'm not able to add comments to the lines below. Can you please make the following change?

Line 13:
Specifies the type of UI used to select date or time. This demo shows how to change calendar to roller date picker.

Can we update this line as follows?

`Specifies the type of UI used to select a date or time. This demo shows how to change a calendar to a roller date picker.`

Also, is "roller date picker" the correct technical term? Just checking.

File: JSDemos/Demos/DateBox/Overview/description.md [hunk 3]
```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
 A value the DateBox displays.
 
 - [type](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#type)       
-Specifies whether the DateBox allows users to select only date, only time, or both.
+Specifies whether the DateBox allows users to select one of the following types:
+
+  - *"date"*    
+  Users can select the date from the calendar, or they can type in their own values.
```
Comment (albertov05) [inline JSDemos/Demos/DateBox/Overview/description.md]: I noticed a warning icon is displayed if the date is not input in the correct format. Can we add "(in the required format)" at the end of this sentence?

`Users can select the date from the calendar, or they can type in their own date value (in the required format).`

File: JSDemos/Demos/DateBox/Overview/description.md [hunk 4]
```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
...
+
+  - *"time"*    
+  Users can select from the time slots that lay in range from 12 AM to 11:30 PM with a step of 30 minutes, or they can type in their own values. Specify the [interval](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#interval) property to set the different time step.
```
Comment (albertov05) [inline JSDemos/Demos/DateBox/Overview/description.md]:
```suggestion
  Users can select a time from a range between 12:00 AM and 11:30 PM at 30-minute intervals, or they can type in their own time value (in the required format). Specify the [interval](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#interval) property to set the time interval.
```

File: JSDemos/Demos/DateBox/Overview/description.md (hunk 1 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/DateBox/Overview/description.md]: Yes, I'll change it in the next commit, that's a great suggestion.

File: JSDemos/Demos/DateBox/Overview/description.md (hunk 2 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/DateBox/Overview/description.md:19]: Yes, it's a correct term. I will update this in the next commit.

File: JSDemos/Demos/DateBox/Overview/description.md (hunk 3 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/DateBox/Overview/description.md]: Noted.

File: JSDemos/Demos/DateBox/Overview/description.md (hunk 1 above)
Comment (albertov05) [reply JSDemos/Demos/DateBox/Overview/description.md]: Thank you.

File: JSDemos/Demos/DateBox/Overview/description.md (hunk 2 above)
Comment (albertov05) [reply JSDemos/Demos/DateBox/Overview/description.md:19]: Thank you.

File: JSDemos/Demos/DateBox/Overview/description.md (hunk 3 above)
Comment (albertov05) [reply JSDemos/Demos/DateBox/Overview/description.md]: Thanks.

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**